nominal/core/channel.py

```diff
@@ -214,70 +215,26 @@ def _decimate_request(
         response = self._clients.compute.compute(self._clients.auth_header, request)
         return response
 
-    def _get_series_values_csv(
-        self,
-        start: api.Timestamp,
-        end: api.Timestamp,
-        relative_to: datetime | IntegralNanosecondsUTC | None = None,
-        relative_resolution: _LiteralTimeUnit = "nanoseconds",
-        *,
-        enable_gzip: bool = True,
-    ) -> BinaryIO:
-        """Get the channel data as a CSV file-like object.
+    def _to_compute_series(self, tags: Mapping[str, str] | None = None) -> scout_compute_api.Series:
+        if tags:
+            series_tags = {key: scout_compute_api.StringConstant(literal=value) for key, value in tags.items()}
+        else:
+            series_tags = {}
```

Comment:
we should add a lint rule for this, but we should not conditionally define variables